src/Lecture1.hs

```diff
@@ -81,7 +87,14 @@ and 1 is the smallest, and 7 - 1 = 6.
 Try to use local variables (either let-in or where) to implement this
 function.
 -}
-minmax x y z = error "TODO"
+
+minmax :: Int -> Int -> Int -> Int
+minmax x y z = 
+    let
+        low = min x (min y z) -- Searched on the internet and wanted to add different options.
+        high = maximum [x, y, z]
```

Comment:
That's a very nice implementation 💎 
Yes, both options with `min/max` and `minimum/maximum` look great 👍🏻 

I would recommend a version with `min/max` because it doesn't allocate a new list. Also, functions `maximum` and `minimum` are considered not that great because they throw errors on empty lists. It's not a problem in this particular case though as it's guaranteed that the list is not empty 🙂